build/critical.go

```diff
@@ -13,4 +14,5 @@ func Critical(v ...interface{}) {
 		panic(s)
 	}
 	os.Stderr.WriteString(s)
+	debug.PrintStack()
```

Comment:
Are we keeping this in or was it just for testing this PR?  Probably useful generally